portfolio/src/main/webapp/comment-form.js

```diff
@@ -1,9 +1,20 @@
-/** Retrieve and post user comments */
+/** Retrieve and post the default quantity of user comments */
 async function getComments() {
-  const response = await fetch('/handle-comment');
-  const comments = await response.json();
+  postFeed(3);
```

Comment:
Make the default value a constant, so it's not a magic number and has some context attached to it.

```javascript
const DEFAULT_COMMENT_COUNT = 3;
```

Reply:
adjusted